src/js/helpers/GoogleAnalyticsPageView.js

```diff
@@ -0,0 +1,13 @@
+import ReactGA from 'react-ga'
+
+const GoogleAnalyticsPageView = () => {
+  ReactGA.set({
+    page: window.location.pathname,
+    location: `${window.location.origin}${window.location.pathname}${window.location.search}`
```

Comment:
since I'm a GA dummy, can you explain why we set a page and location separately?

Reply:
Not a dummy at all -- these two values populate the dl and dp variables that are sent back to google, and I noticed that dl was not updating with pushState changes -- only with full page loads. The issue I'm following in the react-ga repo, I mentioned above, talks about this some.
Here's some info [GA params](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters)